dotcom-rendering/src/components/GridItem.tsx

```diff
@@ -35,6 +36,26 @@ const gridAreaStyles = (area: string) => {
 		`;
 	}
 
+	if (area === 'title') {
+		return css`
+			grid-area: ${area};
+			.sticky-tag-link-test & {
```

Comment:
We only want to apply these styles to elements whose ancestors have class name - this prevents us from having to prop drill the test through to grid item.